compiler/src/iree/compiler/Dialect/Encoding/IR/EncodingInterfaces.td

```diff
@@ -89,6 +89,19 @@ def IREEEncoding_EncodingLayoutAttrInterface :
         assert(false && "unimplemented interface method");
         return {};
       }]
+    >,
+    InterfaceMethod<
+      /*desc=*/[{
+        Returns the layout list.
+      }],
+      /*retTy=*/"::mlir::ArrayAttr",
+      /*methodName=*/"getLayouts",
+      /*args=*/(ins),
+      /*methodBody=*/"",
+      /*defaultImplementation=*/[{
+        assert(false && "unimplemented interface method");
+        return {};
+      }]
```

Comment:
I think we do not need a new interface method. The `getLayout` should already serve the needs.